pkg/rsyslog/setup-rsyslog.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+git clone -b v8-stable https://github.com/rsyslog/rsyslog.git
+cd rsyslog || exit
+mkdir -p contrib/imemlogd
+cp ../imemlogd.diff imemlogd.diff
+git apply imemlogd.diff
```

Comment:
Can't these two lines be turned into a single git apply ?

Reply:
merged them to a single line.